pages/features.html

```diff
@@ -6,104 +6,134 @@
     </p>
 
     <p><b>{% trans "Beat Looping" %}</b><br>
-      {% trans "Found a sweet loop or need to extend your mix while you prep the next track? Instantly loop a 4, 8, or 16 beat segment with the press of a button." %}
+      {% trans "Need to extend your mix while you bring in the next track? Loop a 4, 8, or 16 beat segment with ease." %}
     </p>
 
-    <p><b>{% trans "Powerful Master Sync" %}</b><br>
+    <p><b>{% trans "Master Sync" %}</b><br>
       {% trans "Engage master sync on your decks and they'll stay locked in time even if you change speed.  Construct remixes on the fly with multiple tracks and loops without losing control." %}
     </p>
 
     <p><b>{% trans "Hotcues" %}</b><br>
-      {% trans "Four hotcue points can be dropped or triggered with ease. Our playback engine is finely tuned for accurate, rapid fire hotcue triggering so you can mash and remix as hard as you want to without a hiccup." %}
+      {% trans "Set hotcues to mark places in tracks. Remix and mash up tracks with accurate, rapid fire hotcue triggering." %}
     </p>
   </div>
-
   <div class="halfbox_right">
     <img width="362px" src="{% static '/static/images/feature_waveform-2.0.png' %}">
     <p><b>{% trans "Beat Rolls and Censor" %}</b><br>
-      {% trans "Play with rhythm by triggering short loops and reverse-playback effects.  The track stays in time so you don't miss a beat." %}
+      {% trans "Play with rhythm by triggering short loops and reverse-playback effects. The track stays in time so you don't miss a beat." %}<!--TODO: Write a better way of explaining slip mode.-->
     </p>
 
-    <p><b>{% trans "Intuitive Pitchbend" %}</b><br>
-      {% trans "Ramping pitchbend ensures any tempo nudging is smoothly applied, just like with vinyl." %}
+    <p><b>{% trans "Quantization" %}</b><br>
+      {% trans "Mark and activate cue points and loops exactly on beat every time." %}
     </p>
 
     <p><b>{% trans "Broad Format Support" %}</b><br>
-      {% trans "Our mature audio decoders deliver pristine sound with rock-solid stability. Mixxx plays MP3, M4A/AAC, Ogg Vorbis, Opus, FLAC, WAVE, and AIFF formats." %}
+      {% trans "Compatibile with music files in lossless FLAC, WAV, and AIFF formats as well as lossy MP3, M4A/AAC, Ogg Vorbis, and Opus formats." %}
     </p>
 
     <p><b>{% trans "EQ and Crossfader Control" %}</b><br>
-      {% trans "Mixxx features multiple choices of audiophile filters with adjustable EQ shelves. Crossfader curve control lets you deliver quick cuts or long, smooth crossfades" %}.
+      {% trans "Choose between multiple audiophile filters with adjustable EQ shelves. Crossfader curve control lets you deliver quick cuts or long, smooth crossfades." %}
```

Comment:
Looking at this again, I'd like to remove the word "audiophile". It comes with baggage. To some DJs, "audiophile" has a connotation with spending $1000 on gold cables for no reason, and I think conflating Mixxx with that could turn some people off.